packages/apollo-server-lambda/src/lambdaApollo.ts

```diff
@@ -42,7 +38,7 @@ export function graphqlLambda( options: GraphQLOptions | LambdaGraphQLOptionsFun
       gqlResponse = await runHttpQuery([event, lambdaContext], {
         method: event.httpMethod,
         options: options,
-        query: query,
+        query: JSON.stringify(query),
```

Comment:
I'm not sure about this, but it was the only way to make typescript compiles as it wants the query to be a string.

Reply:
In that case, rather than doing this, we can just remove the `JSON.parse` occurring on [line 45 above](https://github.com/apollographql/apollo-server/pull/730/files#diff-ffe6345b551459399aaa8e1038051488R45), right?